bindings/python/examples/dnn/dnn.py

```diff
@@ -107,11 +107,6 @@ class ModesEnum(Enum):
     status = camera1.setMode(modes[ModesEnum.MODE_NEAR.value])
```

Comment:
This one is using a deprecated modesEnum structure. Needs to be updated as well.